src/mk.sh

```diff
@@ -525,6 +525,14 @@ function mk_build()
   IFS=' ' read -r -a options <<< "$raw_options"
   for option in "${options[@]}"; do
     case "$option" in
+      --info|-i)
```

Comment:
Let's capture the kernel version:
make kernelversion

Reply:
Should the kernel version be a new information provided alongside the kernel release and the total number of compiled modules or should it replace the kernel release?

Reply:
Yes